.github/workflows/Build.yml

```diff
@@ -0,0 +1,45 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Setup dotnet 1.1
+      uses: actions/setup-dotnet@v1
+      with: 
+        dotnet-version: 1.1.14
+
+    - name: Setup dotnet 3.1
+      uses: actions/setup-dotnet@v1
+      with: 
+        dotnet-version: 3.1.202
+
+    - name: Setup dotnet 5.0
+      uses: actions/setup-dotnet@v1
+      with: 
+        dotnet-version: 5.0.100-preview.3
+
+    - name: Setup NuGet
+      uses: nuget/setup-nuget@v1
+      with:
+        nuget-version: '5.x'
+
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1.0.0
+
+    - name: Add nuget source
+      run: dotnet nuget add source -n dotnet5 https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet5/nuget/v3/index.json
```

Comment:
Why are you adding this?

Reply:
@twsouthwick Sorry, this is the copy code from my other repo